ietf/templates/meeting/session_agenda_include.html

```diff
@@ -1,5 +1,5 @@
 {# Copyright The IETF Trust 2015, All Rights Reserved #}
-{# expects slug, session, and timeslot  to be in the context #}
+{# expects slug, session, and timeslot  to be in the context. Calling template must load the agenda_materials.js script. #}
```

Comment:
Good catch. It turns out both of those places leave out the `show_agenda` parameter to the template, which is as if it were `False`. In that case, the modal button is not rendered in the HTML and the JS is not needed. Everything works as it should.

Rather than including unused JS, I will add comments explaining this in the relevant places. I'm also explicitly adding `show_agenda=False` in the places where the templates are included to minimize the risk that someone changes that flag and inadvertently breaks the pages.

Reply:
Correction: one place left out the `show_agenda`, the other already did explicitly set it to `False`.